pkg/kubecost/kubecost_codecs.go

```diff
@@ -13,12 +13,11 @@ package kubecost
 
 import (
 	"fmt"
+	util "github.com/opencost/opencost/pkg/util"
```

Comment:
Goland strikes again

Reply:
@mbolt35 , that other PR I have for bingen fixes this and uses the gofmt standard (group by first/second/third party imports)